apps/web/src/components/studio/editor.tsx

```
@@ -0,0 +1,97 @@
+      <div className="flex flex-col py-16">
+        <div className="flex flex-col gap-4">
+          {sortedBlocks.map((block) => (
+            <Block key={block.id} block={block} />
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function Block({ block }: { block: Block & { children: Node[] } }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Block'. Consider to delete it or rename it.

'Block' is defined here:


(lint/suspicious/noRedeclare)